bundles/org.openhab.binding.bmwconnecteddrive/README.md

```diff
@@ -0,0 +1,686 @@
+| password        | text    | BMW ConnectedDrive Password                                       |
+| region          | text    | Select region in order to connect to the appropriate BMW server.   |
+
+The region Configuration has 3 different options
+
+* _NORTH_AMERICA_
+* _CHINA_
+* _ROW_  (Rest of World)
+
+### Thing Configuration
+
+Same configuration is needed for all things
+
+| Parameter       | Type    | Description                           |           
+|-----------------|---------|---------------------------------------|
+| vin             | text    | Vehicle Identification Number (VIN)   |
+| refreshInterval | integer | Refresh Interval in Minutes           |
+| units           | text    | Unit Selection. See below.            |
+| imageSize       | integer | Image Size                            |
+| imageViewport   | text    | Image Viewport                        |
+
+The unit configuration has 3 options
+
+* _AUTODETECT_ selects miles for US & UK, kilometer otherwise
+* _METRIC_ selects directly kilometers
+* _IMPERIAL_ selects directly miles
+
+The _imageVieport_ allows to show the vehicle from different angels.
+Possible options are 
+
+* _FRONT_
+* _REAR_
+* _SIDE_
+* _DASHBOARD_
+* _DRIVERDOOR_
+
+## Channels
+
+There are many channels available for each vehicle. 
+For better overview they are clustered in different channel groups.
+They differ for each vehicle type, build-in sensors and activated services.
+
+
+### Thing Channel Groups 
+
+#### Vehicle Status
+
+Reflects overall status of the vehicle.
+
+* Channel Group ID is **status**
+* Available for all vehicles
+* Read-only values
+
+| Channel Label             | Channel ID          | Type          | Description                                    |
+|---------------------------|---------------------|---------------|------------------------------------------------|
+| Overall Door Status       | doors               | String        | Combined status for all doors                  |
+| Overall Window Status     | windows             | String        | Combined status for all windows                |
+| Doors Locked              | lock                | String        | Status if doors are locked or unlocked         |
+| Next Service Date         | service-date        | DateTime      | Date of upcoming service                       |
+| Mileage till Next Service | service-mileage     | Number:Length | Mileage till upcoming service                  |
+| Check Control             | check-control       | String        | Presence of active warning messages            |
+| Charging Status           | charge              | String        | Only available for phev, bev_rex and bev       |
+| Last Status Timestamp     | last-update         | DateTime      | Date and time of last status update            |
+
+Overall Door Status values
+
+* _Closed_ - all doors closed
+* _Open_ - at least one door is open
+* _Unknown_ - no door data delivered at all
```

Comment:
You could set the Channel to UNDEF in this case. Same for below.

Reply:
I changed all occurences of Unknown to Undef